src/react/components/window/SidePanel.jsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+import { Message, Confirm } from 'oskari-ui';
+import { CloseCircleFilled } from '@ant-design/icons';
+
+const BUNDLE_KEY = 'oskariui';
+
+const StyledPanel = styled('div')`
+    background: #FFF;
+    position: relative;
+    height: 100%;
+    top: 0;
+    /* sidebar has 3, we want to open it on top of this */
+    z-index: 2;
+    width: 252px;
+    display: flex;
+    flex-direction: column;
+
+    div.header {
+        background-color: #FDF8D9;
+        padding: 5px 10px;
+        div.icon-close {
+            float: right;
+        }
```

Comment:
This can be removed as we are using the AntD CloseCircleFilled icon instead of the older png close icon.